examples/golang/components/http-client-tinygo/wadm.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: http-client-tinygo
+  annotations:
+    version: v0.0.1
+    description: "HTTP client world demo in Golang (TinyGo), using the WebAssembly Component Model and WebAssembly Interfaces Types (WIT)"
+spec:
+  components:
+    - name: http-component
+      type: component
+      properties:
+        image: file://./build/http_client_tinygo_s.wasm
+      traits:
+        # Govern the spread/scheduling of the component
+        - type: spreadscaler
+          properties:
+            replicas: 1
```

Comment:
Here I think we need to add the http client capability provider and a link to it to make it work in wasmCloud

Reply:
done